[2023-06-30T10:23:00.375+0100] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cnt_to_db.create_postgres_table manual__2023-06-30T09:17:24.660804+00:00 [queued]>
[2023-06-30T10:23:00.385+0100] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cnt_to_db.create_postgres_table manual__2023-06-30T09:17:24.660804+00:00 [queued]>
[2023-06-30T10:23:00.386+0100] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2023-06-30T10:23:00.402+0100] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2023-06-30 09:17:24.660804+00:00
[2023-06-30T10:23:00.405+0100] {standard_task_runner.py:57} INFO - Started process 132669 to run task
[2023-06-30T10:23:00.409+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cnt_to_db', 'create_postgres_table', 'manual__2023-06-30T09:17:24.660804+00:00', '--job-id', '3', '--raw', '--subdir', '/home/haboubi/Desktop/api/API_weather/dags/__pycache__/dag_connect_db.py', '--cfg-path', '/tmp/tmpx7_4lyuz']
[2023-06-30T10:23:00.411+0100] {standard_task_runner.py:85} INFO - Job 3: Subtask create_postgres_table
[2023-06-30T10:23:00.489+0100] {task_command.py:410} INFO - Running <TaskInstance: cnt_to_db.create_postgres_table manual__2023-06-30T09:17:24.660804+00:00 [running]> on host haboubi-VivoBook-15-ASUS-Laptop-X540UB
[2023-06-30T10:23:00.620+0100] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='haboubi' AIRFLOW_CTX_DAG_ID='cnt_to_db' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2023-06-30T09:17:24.660804+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-30T09:17:24.660804+00:00'
[2023-06-30T10:23:00.621+0100] {sql.py:265} INFO - Executing: 
                create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt,dag_id)

                )
                
                
                
[2023-06-30T10:23:00.642+0100] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-30T10:23:00.679+0100] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-30T10:23:00.693+0100] {sql.py:375} INFO - Running statement: 
                create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt,dag_id)

                )
                
                
                , parameters: None
[2023-06-30T10:23:00.712+0100] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=cnt_to_db, task_id=create_postgres_table, execution_date=20230630T091724, start_date=20230630T092300, end_date=20230630T092300
[2023-06-30T10:23:00.742+0100] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-30T10:23:00.777+0100] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
